packages/react/src/checkout/hooks/useCheckout.ts

```diff
@@ -476,6 +481,42 @@ function useCheckout(
     reset();
   }
 
+  const {
+    data: { countryAddressSchemas },
+    actions: { fetch: fetchCountryAddressSchemas },
+  } = useCountryAddressSchemas(
+    (checkoutOrder?.shippingAddress?.country.alpha2Code as string) || '',
+    { enableAutoFetch: false },
+  );
+
+  const isShippingAddressZipCodeValid = useCallback(
+    async (shippingAddress?: CheckoutAddress) => {
+      let addressSchema;
+
+      addressSchema = countryAddressSchemas;
+
+      const tempShippingAddress =
```

Comment:
You can remove this line and add use the default value for the shippingAddress parameter like this:

```js
async (shippingAddress: CheckoutAddress = checkoutOrder?.shippingAddress) => {
```

Reply:
updated, pls double check